### DISCUSSION TITLE
Sunrise/Sunset time

### DISCUSSION THREAD

**Discussion by redfusion83** [basnijholt/adaptive-lighting#1055] [Q&A]
Sorry, this may be a silly question. I'm assuming that if I don't enter anything in the configuration about sunrise and sunset times, the adaptive lighting will pull this information from the sun.sun integration. However, I haven't been able to confirm this, so .... can anyone tell me if this is correct? Thanks in advance!

**Comment by fernbrun** [comment]
Same question here

**Comment by beasthouse-au** [comment]
Looks like the code references `homeassistant.helpers.sun` so I think its a safe bet that it does use the Sun Integration in lieu of manual overrides.
